vllm/hpu/ops.py

```diff
@@ -253,41 +253,64 @@ def forward(self, state):
 
 class StaticFusedMOE(torch.nn.Module):
 
-    def __init__(self, num_total_experts):
+    def __init__(self, num_total_experts, using_inc_quantization):
```

Comment:
its not really controlling wether to use inc or not. this is just an artifact. its controlling dynamic vs. static moe, so please rename.

Reply:
I updated to use_static_implementation. I also changed StaticFusedMOE class name to FusedMOE since we have static and dynamic paths now.